.travis.yml

```diff
@@ -6,6 +6,11 @@ python:
   - "3.8"
   - "3.9"
 
+before_install:
+  - pip install black
+  - echo "Black formatting check"
+  - black --check *.py
```

Comment:
Can you make this `-l 100` there too?
100 chars per line is better these days

Reply:
> Can you make this `-l 100` there too? 100 chars per line is better these days

Yep, on it.